api/src/main/resources/public/api/v0/schema/jobspec_v0.schema.json

```diff
@@ -250,9 +250,14 @@
                 "type": "string",
                 "description": "Possible values are RO for ReadOnly and RW for Read/Write",
                 "enum": ["RO", "RW"]
+              },
+              "secret": {
+                "type": "string",
+                "description": "References the secret that is used by this volume",
+                "minLength": 1
               }
             },
-            "required": ["containerPath", "hostPath", "mode"]
```

Comment:
Do you plan to follow this up with another pull request to add appropriate validation?